it.sh

```diff
@@ -193,9 +193,7 @@ fi
 
 CMD=$1
 shift
-# Added -Dcyclonedx.skip=true to avoid ISO-8859-1 [ERROR]s
-# May be fixed in the future
-MAVEN_IGNORE="-P skip-static-checks,skip-tests -Dmaven.javadoc.skip=true -Dcyclonedx.skip=true"
+MAVEN_IGNORE="-P skip-static-checks,skip-tests -Dmaven.javadoc.skip=true"
```

Comment:
This one can remain since we don't need to repeat the checks for the ITs. No worries; I'll add it in a different PR.